ClientApp/src/app/log-in/log-in.component.ts

```diff
@@ -19,14 +21,24 @@ export class LogInComponent {
 
  constructor(
    public dialogRef: MatDialogRef<LogInComponent>,
-   private userService: UserService) {}
+   public userService: UserService) {}
```

Comment:
Why did you change this to public?

Reply:
I use this service in HTML file for this component to check if user is logged in. If they are private I don't have access to them